cmd/cloud/key.go

```diff
@@ -83,7 +83,11 @@ func newListKeyCommand() *cobra.Command {
 		Use:   "list",
 		Short: "List API keys",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := executeListKeys()
+			err := getNewAccessTokenUsingRefreshTokenIfCurrentIsExpired()
+			if err != nil {
+				logger.Errorf("Failed to refresh access token: %v", err)
```

Comment:
We should use `ui.PrintError` here to show the error to the user because this error is a possibly fatal error for the command.